mergify_engine/redis_utils.py

```diff
@@ -162,6 +173,7 @@ def team_permissions_cache(self) -> RedisTeamPermissionsCache:
             config.TEAM_PERMISSIONS_CACHE_URL,
             decode_responses=False,
             max_connections=self.cache_max_connections,
+            redis_connect_func=None,
```

Comment:
this is the default, why setting it?

Reply:
I never found how to please mypy when the method uses @overload and not (re)set the default on every calls.

Reply:
Cool!